cocos/core/event/callbacks-invoker.ts

```diff
@@ -41,16 +42,40 @@ class CallbackInfo {
     public once = false;
 
     public set (callback: Function, target?: Object, once?: boolean) {
-        this.callback = callback;
+        this.callback = callback || empty;
         this.target = target;
         this.once = !!once;
     }
+
+    public reset () {
+        this.target = undefined;
+        this.callback = empty;
+        this.once = false;
+    }
+
+    public check () {
+        // Validation
+        if (this.target instanceof CCObject && !isValid(this.target, true)) {
+            return false;
+        }
+        else {
+            return true;
+        }
+    }
+
+    public execute (arg0?: any, arg1?: any, arg2?: any, arg3?: any, arg4?: any) {
```

Comment:
I don't think execute API is appropriate for `CallbackInfo`. CallbackInfo is more like a set of data.
If we need to invoke the data, why not invoke in CallbacksInvoker?

Reply:
Removed execute